packages/data/src/redux-store/metadata/actions.js

```diff
@@ -32,6 +32,42 @@ export function finishResolution( selectorName, args ) {
 	};
 }
 
+/**
+ * Returns an action object used in signalling that a batch of selector resolutions has
+ * started.
+ *
+ * @param {string} selectorName Name of selector for which resolver triggered.
+ * @param {...*}   args         Array of arguments to associate for uniqueness, each item
+ * 								is associated to a resolution.
+ *
+ * @return {Object} Action object.
+ */
+export function startResolutions( selectorName, args ) {
```

Comment:
Would it be possible to refactor existing actions (`finishResolution` and `finishResolution`) to accept not only a single resolution but also multiple resolutions when necessary? This way we wouldn't have to create new methods.

Reply:
How would we determine what the caller intends to do? `args` is already an array, and for `finishResultions` it's an array of arrays.

In my view two separate functions are easier to understand. And easier to provide types for.

Reply:
I thought about that option but I didn't want to overuse the `args` parameter, besides since args can be anything it can be hard to infer when it's being dispatched with single/multiple resolutions. One option we might consider is to add an extra boolean parameter that specifies if the args contain single/multiple resolutions.

Reply:
Right, it's very hard to detect whether it's single or multiple items because `args` can be any number of params of any type.

In `startResolutions` and `finishResolutions` we shouldn't use a different name than `args` to distinguish between single and multiple items in both functions. The type is going to be more complex as well.

I think it's better to add new functions if the only alternative is a boolean param.